data/types/bolus/prescriptor/prescriptor_test.go

```diff
@@ -0,0 +1,105 @@
+package prescriptor_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	dataNormalizer "github.com/tidepool-org/platform/data/normalizer"
+	"github.com/tidepool-org/platform/data/types/bolus/prescriptor"
+	dataTypesTest "github.com/tidepool-org/platform/data/types/test"
+	errorsTest "github.com/tidepool-org/platform/errors/test"
+	"github.com/tidepool-org/platform/pointer"
+	"github.com/tidepool-org/platform/structure"
+	structureValidator "github.com/tidepool-org/platform/structure/validator"
+	"github.com/tidepool-org/platform/test"
+)
+
+func NewPrescriptor() *prescriptor.Prescriptor {
+	datum := prescriptor.NewPrescriptor()
+	datum.Prescriptor = pointer.FromString(test.RandomStringFromArray(prescriptor.Presciptors()))
+	return datum
+}
+
+func ClonePrescriptor(datum *prescriptor.Prescriptor) *prescriptor.Prescriptor {
+	if datum == nil {
+		return nil
+	}
+	clone := prescriptor.NewPrescriptor()
+	clone.Prescriptor = pointer.CloneString(datum.Prescriptor)
+	return clone
+}
```

Comment:
Duplicates functions of  data/types/bolus/prescriptor/test/prescriptor.go

Reply:
same as above